recipes/torchmanager/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "torchmanager" %}
+{% set version = "1.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchmanager-{{ version }}.tar.gz
+  sha256: edb6b2063968d1a740b6a0bc591242986c141f06ed582ff176035ac77786c955
+
+build:
+  noarch: python
+  skip: true
```

Comment:
```suggestion
```

Reply:
Windows check will not be passed if skip is not given due to the lack of PyTorch for the platform in conda-forge.

Reply:
That's fine for `noarch: python` recipes. With this line though, everything will be skipped and CI does nothing.

Reply:
Please take a look at the Please take a look at the Windows error, saying that pytorch does not exist